app/controllers/application_controller.rb

```diff
@@ -4,8 +4,18 @@ class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
   before_action :set_app_info_headers
 
+  private
+
   def set_app_info_headers
     headers["X-GitHub-Repository"] = "https://github.com/department-of-veterans-affairs/vets-api"
```

Comment:
So I'm only raising this one out of curiosity, what exactly is this for and why is it necessary for the header? Is this a roadrunner artifact? I think we should look at calling it Vets Api V1 or something, and use numbered versioning vs hashed based X-Git-SHA for the revision?